docs/fiddles/screen/fit-screen-external/main.js

```diff
@@ -0,0 +1,24 @@
+// Creating a window in the external display
+//
+// For more info, see:
+// https://electronjs.org/docs/api/screen
+
+const electron = require('electron')
+const { app, BrowserWindow } = require('electron')
```

Comment:
We shouldn't `require('electron')` twice

```suggestion
const { app, screen, BrowserWindow } = require('electron')
```

Reply:
Hey ckerr, 
We can't require screen before the app is ready. If you have other suggestions lmk! 😄

Reply:
d'oh, you're right. :+1: on marking this resolved.